vendor.conf

```diff
@@ -3,6 +3,7 @@
 # using.
 github.com/opencontainers/runtime-spec 5684b8af48c1ac3b1451fa499724e30e3c20a294
 # Core libcontainer functionality.
+github.com/checkpoint-restore/go-criu master
```

Comment:
can you pin to a specific commit or tag and not master?

Reply:
Good point. When I initially made this PR we did not have tags in go-criu, but other projects have asked for the same. Now that we have tags (which are following the CRIU releases) I changed it to v3.11